src/cohere/client.py

```diff
@@ -185,17 +197,107 @@ def embed(
     delete_connector: Never = moved_function("delete_connector", ".connectors.delete")
     oauth_authorize_connector: Never = moved_function("oauth_authorize_connector", ".connectors.o_auth_authorize")
 
+    def tokenize(
+        self,
+        *,
+        text: str,
+        model: typing.Optional[str] = OMIT,
+        request_options: typing.Optional[RequestOptions] = None,
+        offline: bool = True,
+    ) -> TokenizeResponse:
+        """
+        This function splits input text into smaller units called tokens using byte-pair encoding (BPE). To learn more about tokenization and byte pair encoding, see the tokens page.
+
+        Parameters:
+            - text: str. The string to be tokenized, the minimum text length is 1 character, and the maximum text length is 65536 characters.
+
+            - model: typing.Optional[str]. An optional parameter to provide the model name. This will ensure that the tokenization uses the tokenizer used by that model.
+
+            - request_options: typing.Optional[RequestOptions]. Request-specific configuration.
+
+            - offline: typing.Optional[bool]. An optional parameter to specify whether to use an offline tokenizer. If set to True, the tokenizer config will be downloaded (and cached),
+                and the request will be processed using the offline tokenizer. If set to False, the request will be processed using the API. The default value is True.
+        ---
+        from cohere.client import AsyncClient
 
-class AsyncClient(AsyncBaseCohere):
+        client = Client(
+            client_name="YOUR_CLIENT_NAME",
+            token="YOUR_TOKEN",
+        )
+        client.tokenize(
+            text="tokenize me! :D",
+            model="command",
+        )
+        """
+        # We cannot make the parameter required in the signature because otherwise the inheritance will break (mypy incompatible with supertype)
+        if model is OMIT:
+            raise ValueError("The model parameter is required.")
```

Comment:
can we fix this in the spec so we can fix the types?

Reply:
Yeah but it has to be done on the endpoint. I can do that now instead ✅

Reply:
done